src/main/groovy/com/vanniktech/android/junit/jacoco/JunitJacocoExtension.groovy

```diff
@@ -28,4 +28,19 @@ class JunitJacocoExtension {
      * @since 0.6.0
      */
     boolean includeNoLocationClasses
+
+    /**
+     * Whether or not to use this plugin for configuring the Jacoco version and creating
+     * a task to combine the coverage reports for instrumentation and unit tests. The default
+     * value is true.
+     * @since 0.13.0
+     */
+    boolean configureInstrumentationCoverage = true
+
+    /**
+     * Whether or not to include instrumentation coverage in the final global merged report.
+     * Note that this will run all instrumentation tests when true.
+     * @since 0.13.0
+     */
+    boolean includeInstrumentationCoverageInMergedReport = false
```

Comment:
Since this can be really heavy, I thought it's safer to turn this off by default

Reply:
Yup agreed. I also think the majority won't need a single report